utils/test.key

```diff
@@ -0,0 +1,27 @@
+-----BEGIN RSA PRIVATE KEY-----
```

Comment:
nit: can we put these keys in the `fixtures` directory?  Also, you may not need this key without the role header because we other keys (ex: `secure.example.com.key`) without headers that we can test with